include/Reader/readerir.h

```diff
@@ -1139,6 +1143,18 @@ class GenIR : public ReaderBase {
     return makeLoad(Address, IsVolatile, false);
   }
 
+  /// \brief Create a call or invoke instruction
+  ///
+  /// The call is inserted at the LLVMBuilder's current insertion point.
+  ///
+  /// \param Callee    Target of the call
+  /// \param Args      Arguments to pass to the callee
+  /// \param MayThrow  True if the callee may raise an exception
+  /// \returns         A \p CallSite wrapping the CallInst or InvokeInst
+  llvm::CallSite makeCall(llvm::Value *Callee,
+                          llvm::ArrayRef<llvm::Value *> Args,
+                          bool MayThrow = true);
```

Comment:
Good catch, thanks.  I'd actually started with defaults and then decided it was a bad idea and switched the helper call stuff to no default, but forgot to go the next level down and get rid of the default on makeCall.  I've updated to remove the default on makeCall and push the parameterization upstream to where we can reasonably make the decision (though in the case of ReaderBase::rdrCall I actually didn't spot any intrinsics that I was sure are safe...).